vendor/bat-native-ads/src/bat/ads/internal/ad_targeting/processors/behavioral/bandits/epsilon_greedy_bandit_processor.cc

```diff
@@ -32,23 +32,23 @@ EpsilonGreedyBanditArmMap MaybeAddOrResetArms(
 
   for (const auto& segment : kSegments) {
     const auto iter = updated_arms.find(segment);
-
     if (iter != updated_arms.end()) {
-      BLOG(3, "Epsilon greedy bandit arm already exists for " << segment
-                                                              << " segment");
+      const EpsilonGreedyBanditArmInfo arm = iter->second;
+      if (!arm.IsValid()) {
+        EpsilonGreedyBanditArmInfo arm;
+        arm.value = kArmDefaultValue;
+        arm.pulls = kArmDefaultPulls;
 
-      continue;
-    }
+        updated_arms[segment] = arm;
```

Comment:
@moritzhaller If the arm provided in the JSON is invalid should we not just skip this arm?